ArcGISRuntimeSDKQt_SampleViewers/ArcGISRuntimeSDKQt_CppSamples/zlib_minizip_ng.pri

```diff
@@ -0,0 +1,54 @@
+ # Copyright 2024 Esri.
+
+ # Licensed under the Apache License, Version 2.0 (the "License");
+ # you may not use this file except in compliance with the License.
+ # You may obtain a copy of the License at
+ # http://www.apache.org/licenses/LICENSE-2.0
+
+ # Unless required by applicable law or agreed to in writing, software
+ # distributed under the License is distributed on an "AS IS" BASIS,
+ # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ # See the License for the specific language governing permissions and
+ # limitations under the License.
+
+
+INCLUDEPATH += \
+    $$PWD/zlib-ng \
+    $$PWD/minizip-ng
+
+HEADERS += \
+    $$files($$PWD/zlib-ng/*.h)) \
+    $$PWD/minizip-ng/zip.h \
+    $$PWD/minizip-ng/unzip.h \
+    $$PWD/minizip-ng/ioapi.h \
+    $$PWD/ZipHelper.h
+
+SOURCES += \
+    $$files($$PWD/zlib-ng/*.c)) \
```

Comment:
@JamesMBallard Moved the zlib/minizip/helper related files into this .pri import. I still wanted to include the wildcard for `zlib-ng` is the above correct?

Reply:
Yes the wildcard is fine if you want to pick up a bunch of files like that.

Reply:
For now I want to pick up all zlib-ng files. Ill start working my way through them to see if I can reduce how many we have.